apps/nextra/pages/en/build/indexer/common-queries/account-transactions-gt.mdx

```diff
@@ -0,0 +1,36 @@
+import { GraphQLEditor } from 'components/index'
+import { Callout } from 'nextra/components'
+
+# Get Recent Account Transactions
+
+`GetAccountTransactionsDataGt` gets transactions impacting a specific account that are at least as recent as a given transaction version. This can be useful for only checking the most recent transactions. 
+
+<Callout type="info">
+Try it yourself! You can customize the variables at the bottom of the editor.
+</Callout>
+
+<GraphQLEditor
+  query={`query GetAccountTransactionsDataGt($address: String, $limit: Int, $gt: bigint) {
+  address_version_from_move_resources(
```

Comment:
Are you sure this is the right query? This doesn't look right for getting an accounts transactions... Also it keeps timing out so we should use another query instead

Reply:
This is the right query. It's a variant of the `Account Transactions` request. I updated the example to use the "dead wallet" address which has less updates. (the 0x00...001 address changes so often that the $gt value quickly becomes outdated, resulting in a long query time as many more results are being included I believe). 

I merged this into the `Account Transactions` page as a variant explained at the bottom to simplify the sidebar navigation.

Reply:
Both of them get the transactions which impact an account - not necessarily the ones that the account issued I believe. They are #3 and #8 in most commonly used though.